diff --git a/node_modules/@uniswap/universal-router-sdk/dist/universal-router-sdk.cjs.development.js b/node_modules/@uniswap/universal-router-sdk/dist/universal-router-sdk.cjs.development.js
index d0e1f6e..cc062d3 100644
--- a/node_modules/@uniswap/universal-router-sdk/dist/universal-router-sdk.cjs.development.js
+++ b/node_modules/@uniswap/universal-router-sdk/dist/universal-router-sdk.cjs.development.js
@@ -246,6 +246,10 @@ var CHAIN_CONFIGS = (_CHAIN_CONFIGS = {}, _CHAIN_CONFIGS[1] = {
   router: '0x198EF79F1F515F02dFE9e3115eD9fC07183f02fC',
   weth: '0x4200000000000000000000000000000000000006',
   creationBlock: 1452376
+}, _CHAIN_CONFIGS[1891] = {
+  router: '0x738fD6d10bCc05c230388B4027CAd37f82fe2AF2',
+  weth: '0x7EbeF2A4b1B09381Ec5B9dF8C5c6f2dBECA59c73',
+  creationBlock: 0
 }, _CHAIN_CONFIGS);
 var UNIVERSAL_ROUTER_ADDRESS = function UNIVERSAL_ROUTER_ADDRESS(chainId) {
   if (!(chainId in CHAIN_CONFIGS)) throw new Error("Universal Router not deployed on chain " + chainId);
diff --git a/node_modules/@uniswap/universal-router-sdk/dist/universal-router-sdk.esm.js b/node_modules/@uniswap/universal-router-sdk/dist/universal-router-sdk.esm.js
index a85bcc6..8350e26 100644
--- a/node_modules/@uniswap/universal-router-sdk/dist/universal-router-sdk.esm.js
+++ b/node_modules/@uniswap/universal-router-sdk/dist/universal-router-sdk.esm.js
@@ -243,6 +243,10 @@ var CHAIN_CONFIGS = (_CHAIN_CONFIGS = {}, _CHAIN_CONFIGS[1] = {
   router: '0x198EF79F1F515F02dFE9e3115eD9fC07183f02fC',
   weth: '0x4200000000000000000000000000000000000006',
   creationBlock: 1452376
+}, _CHAIN_CONFIGS[1891] = {
+  router: '0x738fD6d10bCc05c230388B4027CAd37f82fe2AF2',
+  weth: '0x7EbeF2A4b1B09381Ec5B9dF8C5c6f2dBECA59c73',
+  creationBlock: 0
 }, _CHAIN_CONFIGS);
 var UNIVERSAL_ROUTER_ADDRESS = function UNIVERSAL_ROUTER_ADDRESS(chainId) {
   if (!(chainId in CHAIN_CONFIGS)) throw new Error("Universal Router not deployed on chain " + chainId);
